src/test/signalpathtest.h

```diff
@@ -163,8 +161,7 @@ class BaseSignalPathTest : public MixxxTest {
     // Use tools/AudioPlot.py to look at the reference file and make sure it
     // looks correct.  Each line of the generated file contains the left sample
     // followed by the right sample.
-    void assertBufferMatchesReference(const CSAMPLE* pBuffer, const int iBufferSize,
-                                   QString reference_title, const double delta=.0001) {
+    void assertBufferMatchesReference(const CSAMPLE* pBuffer, const int iBufferSize, QString reference_title, const double delta = .0001) {
```

Comment:
Aren't you're using our pre-commit hooks?  This needs some linebreaks.

Reply:
I am, all the time. But I did already notice some curious results previously.

Reply:
Fixed in https://github.com/xerus2000/mixxx/pull/5/commits/ddb672c27caa6b99a4a2cda8403f4db7e4ca67a0